controls/anssi.yml

```diff
@@ -1246,6 +1246,7 @@ controls:
     - ensure_gpgcheck_local_packages
     - ensure_redhat_gpgkey_installed
     - ensure_oracle_gpgkey_installed
+    - ensure_almalinux_gpgkey_installed
```

Comment:
You need to make sure that the rule `ensure_almalinux_gpgkey_installed` doesn't get into content for other products than Alma. You will need to explicitly remove it from all profiles that use the `anssi` control file.
Add
```
 - '!ensure_almalinux_gpgkey_installed'
```
to `selections:`  in all `.profile` files listed by ` git grep -l anssi: *.profile`. 

We already do a similar thing for the `ensure_oracle_gpgkey_installed`, for example:

https://github.com/ComplianceAsCode/content/blob/cccf253994e862d470450cc10f8c12fc1fdcac8c/products/rhel9/profiles/anssi_bp28_high.profile#L52

Reply:
Thanks, I added commit https://github.com/ComplianceAsCode/content/pull/12883/commits/5d7d0fe4b4f40df52920ca6baf2d73b6cd1b8576 to address this for anssi and a couple of other profiles where i missed it.